src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import App from './App';
+
+test('renders app name SURVEYOR', () => {
```

Comment:
Should we follow the template:
```js
describe('whatever', () => {
  context('whatever', () => {
    it('renders something', () => {
       // expect
    })
  })
})
```

Reply:
I guess there is no method `context` in `jest` - https://jestjs.io/docs/en/api#methods

Reply:
Ah, yes, thanks for referencing

Reply:
Updated with the reviewed changes.